.github/workflows/post_benchmark_comment.yaml

```
@@ -0,0 +1,108 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Post benchmark comment
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types: [completed]
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit).
+  group: ${{ github.event.workflow_run.pull_requests[0].number || github.sha }}-post-benchmark-comment
+  cancel-in-progress: true
+
+env:
+  PR_CI_STAGE: ${{ github.event.workflow_run.event == 'pull_request' && 'presubmit' || 'postsubmit' }}
+  PR_CI_RUN_ID: ${{ github.event.workflow_run.id }}
+  PR_CI_RUN_ATTEMPT: ${{ github.event.workflow_run.run_attempt }}
+
+jobs:
+  setup:
+    # Only run the workflow if it's triggered from a pull request.
+    if: github.event.workflow_run.event == 'pull_request'
+    runs-on: ubuntu-20.04
+    outputs:
+      should-run: ${{ steps.configure.outputs.should-run }}
+      runner-env: ${{ steps.configure.outputs.runner-env }}
+      runner-group: ${{ steps.configure.outputs.runner-group }}
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
+        with:
+          # We need the parent commit to do a diff
+          fetch-depth: 2
+      - name: "Configuring CI options"
+        id: configure
+        run: |
+          # Just informative logging. There should only be two commits in the
+          # history here, but limiting the depth helps when copying from a local
+          # repo instead of using checkout, e.g. with
+          # https://github.com/nektos/act where there will be more.
+          git log --oneline --graph --max-count=3
+          ./build_tools/github_actions/configure_ci.py
+
+  check_benchmark_job:
+    needs: setup
+    if: needs.setup.outputs.should-run == 'true'
+    runs-on: ubuntu-20.04
+    outputs:
+      job-conclusion: ${{ steps.check.outputs.job-conclusion }}
+    steps:
+      - name: "Checking benchmark processing job"
+        id: check
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BENCHMARK_PROCESS_JOB: process_benchmark_results
+        run: |
+          gh api \
+            -H "Accept: application/vnd.github+json" \
+            --jq ".jobs | map(select(.name==\"${BENCHMARK_PROCESS_JOB}\"))[0] | .conclusion" \
+            "/repos/${GITHUB_REPOSITORY}/actions/runs/${PR_CI_RUN_ID}/attempts/${PR_CI_RUN_ATTEMPT}/jobs" \
+            > conclusion
```

I don't think the `gh --jq` form offers much benefit over piping to jq here and it has limitations, like you can't add arguments and so you've ended up with all this escaping

```suggestion
"/repos/${GITHUB_REPOSITORY}/actions/runs/${PR_CI_RUN_ID}/attempts/${PR_CI_RUN_ATTEMPT}/jobs" \
  | jq --arg job "${BENCHMARK_PROCESS_JOB}" \
  '.jobs | map(select(.name=="$job"))[0] | .conclusion'
            > conclusion
```

It might also be nicer to use a variable rather than a file